test/utils/IntegrationBaseSetup.t.sol

```diff
@@ -93,7 +93,7 @@ contract IntegrationBaseSetup is Test {
 
         prizePool.setDrawManager(address(this));
 
-        claimer = new Claimer(prizePool, 0.0001e18, 1000e18, drawPeriodSeconds, ud2x18(0.5e18));
+        claimer = new Claimer(prizePool, drawPeriodSeconds, ud2x18(0.5e18));
```

Comment:
should we set max fee to 10% for the sake of consistency with production?

Reply:
Yeah, this is just a script for local testing, but fixed it here for consistency: [5a91b94](https://github.com/GenerationSoftware/pt-v5-testnet/pull/17/commits/5a91b943222bd69f1e267bd6ef005626ffaf39ab)